test/read-only-handler.spec.ts

```diff
@@ -304,7 +304,28 @@ describe('ReadOnlyHandler', () => {
             expect( () => {
                 value.foo = '';
             }).toThrow();
-            expect(todos['entry'].foo).toEqual('bar');
+            expect(todos.entry.foo).toEqual('bar');
+        });
+        it('readonly proxy prevents access to any setter in descriptor', () => {
+            const target = new ReactiveMembrane();
+            const todos = {};
+            let value = 0;
+            Object.defineProperty(todos, 'entry', {
+                get() {
+                    return value;
+                },
+                set(v) {
+                    value = v;
+                },
+                configurable: true
+            });
+            const proxy = target.getReadOnlyProxy(todos);
+            const desc = Object.getOwnPropertyDescriptor(proxy, 'entry');
+            const { set, get } = desc;
+            expect(() => {
+                set.call(proxy, 1);
+            }).toThrow();
```

Comment:
The `toThrow` should validate the error message to ensure it's the expected one.